File: Food trust .java
public class FoodTrust {

    static class Restaurant {
        String name;
        double rating;
        int safetyScore;

        Restaurant(String name, double rating, int safetyScore) {
            this.name = name;
            this.rating = rating;
            this.safetyScore = safetyScore;
        }
    }

    public static String recommend(Restaurant restaurant) {

        if (restaurant.safetyScore >= 85) {
            return restaurant.name +
                    " has strong available safety information.";
        }

        if (restaurant.safetyScore >= 70) {
            return restaurant.name +
                    " has moderate available safety information.";
        }

        return restaurant.name +
                " has limited available safety information.";
    }

    public static void main(String[] args) {

        Restaurant restaurant =
                new Restaurant("Green Bowl", 4.5, 92);

        System.out.println("FoodTrust AI");
        System.out.println(recommend(restaurant));
    }
}

File: Food trust.cpp
#include <iostream>
#include <string>
using namespace std;

class Restaurant {
public:
    string name;
    float rating;
    int safetyScore;

    Restaurant(string n, float r, int s) {
        name = n;
        rating = r;
        safetyScore = s;
    }

    void display() {
        cout << "Restaurant: " << name << endl;
        cout << "Rating: " << rating << endl;
        cout << "Food Safety Confidence: "
             << safetyScore << "/100" << endl;
    }

    string recommendation() {

        if (safetyScore >= 85) {
            return "Strong available safety information.";
        }

        if (safetyScore >= 70) {
            return "Moderate available safety information.";
        }

        return "Limited available safety information.";
    }
};

int main() {

    Restaurant restaurant(
        "Green Bowl",
        4.5,
        92
    );

    restaurant.display();

    cout << "FoodTrust AI: "
         << restaurant.recommendation()
         << endl;

    return 0;
}
